rmarkdowns/Week_1.Rmd

```diff
@@ -226,7 +226,7 @@ The colors we are using here are what are known as "hex colors", because they ar
 ```{r}
 ggplot() + 
   geom_sf(data = world_without_antarctica, 
-          fill = "#669438", color = "#32481B", size = 0.25) +
+          fill="#98FF98", color="#008080", scale = 0.2) +
```

Comment:
Nice colors! I noticed that changing the "size"/"scale" doesn't seem to change the thickness of the line as it should. Still trying to troubleshoot that. But no worries; your map looks good!